plugins/providers/virtualbox/driver/version_5_0.rb

```diff
@@ -587,6 +587,10 @@ def read_guest_additions_version
 
         def read_guest_ip(adapter_number)
           ip = read_guest_property("/VirtualBox/GuestInfo/Net/#{adapter_number}/V4/IP")
+          if ip.end_with?(".1")
+            @logger.warn("VBoxManage guest property returned: #{ip}. Result resembles IP of DHCP server. Retrying IP lookup.")
+            ip = read_guest_property("/VirtualBox/GuestInfo/Net/#{adapter_number}/V4/IP")
```

Comment:
Is this expected to be different than the call on line 589? :thinking:

Reply:
That's sort of my hope... that the `.1` thing is a fluke and the next time you check you get the real value. I haven't been able to reproduce the issue consistently enough to validate that theory. But I think it's a better idea to just bail on `.1` per @chrisroberts's comment.